tools/airgap_linter.py

```diff
@@ -178,7 +180,7 @@ def main(argv):
 
     if invalid:
         print(
-            "Airgap check FAILED. This framework will NOT work in an airgap. Fix the detected issues."
+            "Airgap check FAILED. The framework's working code directory contains references to non-https URL resources and/or there are http URL references made inside marathon/config JSON files. Please consider replacing these resources to their HTTPS equivalent. In case of internal Mesos DNS URLs, please mention http links as URI under resources.json. This framework will NOT work in an airgap. Fix the detected issues."
```

Comment:
> In case of internal Mesos DNS URLs, please mention http links as URI under resources.json

what do you mean? This will not work. All the uris in resource.json needs to be "downloadable" without a running DC/OS cluster.

Reply:
@takirala I may have interpreted it in a different way, but what can we do here to make it sensible? Should we remove this altogether or suggest user to "mention external resources URLs in resources.json"